src/styles/card.scss

```diff
@@ -0,0 +1,39 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
```

Comment:
The width and height of the `.card__image` should generally be defined in percentages or be responsive to maintain the aspect ratio across different screen sizes. Consider using percentages or a responsive design approach unless fixed sizes are a specific requirement.